src/Compiler/CompilerUtils.cpp

```diff
@@ -34,13 +34,21 @@
 #include "src/Conversion/KrnlToLLVM/ConvertKrnlToLLVM.hpp"
 #include "src/Support/OMOptions.hpp"
 
+#include "VCSVersion.inc"
+
 #define DEBUG_TYPE "compiler_utils"
 
 using namespace std;
 using namespace mlir;
 using namespace onnx_mlir;
 
 const string OnnxMlirEnvOptionName = "ONNX_MLIR_FLAGS";
+#if defined(ONNX_MLIR_REPOSITORY) && defined(ONNX_MLIR_REVISION)
+const string OnnxMlirVersion =
+    "onnx-mlir version 1.0.0 (" ONNX_MLIR_REPOSITORY " " ONNX_MLIR_REVISION ")";
```

Comment:
Is it possible to also get the commit for LLVM itself ?  Ideally we want to know the LLVM commit used to build onnx-mlir, as well as the commit for onnx-mlir.